test/.npmrc

```diff
@@ -1 +0,0 @@
-legacy-peer-deps=true
```

Comment:
Did you mean to check this in? I don't remember if this breaks anything -- I realize we want to check npm dependencies in top level apps and the "test app" is such a thing, but we had this for some reason I don't recall. Maybe it's not needed anymore. Maybe I also overlooked this in the other PR updates.

Reply:
Resolved via IM.